website/docusaurus.config.js

```diff
@@ -0,0 +1,168 @@
+      theme: require('prism-react-renderer/themes/github'),
+      darkTheme: require('prism-react-renderer/themes/dracula'),
+    },
+    footer: {
+      style: 'light',
+      links: [
+        {
+          title: 'About',
+          items: [
+            {
+              label: 'What is Pinot?',
+              to: 'https://docs.pinot.apache.org/',
+            },
+            {
+              label: 'Components',
+              to: 'https://docs.pinot.apache.org/pinot-components',
+            },
+            {
+              label: 'Architecture',
+              to: 'https://docs.pinot.apache.org/concepts/architecture',
+            },
+            {
+              label: 'PluginsArchitecture',
+              to: 'https://docs.pinot.apache.org/plugins/plugin-architecture',
+            },
+          ],
+        },
+        {
+          title: 'Components',
+          items: [
+            {
+              label: 'Presto',
+              to: 'https://docs.pinot.apache.org/integrations/presto',
+            },
+            {
+              label: 'PQL',
+              to: 'docs/components/sources',
+            },
+            {
+              label: 'ThirdEye',
+              to: 'https://docs.pinot.apache.org/integrations/thirdeye',
+            },
+            {
+              label: 'PowerBI',
+              to: 'docs/components/sinks',
+            },
+          ],
+        },
+        {
+          title: 'Docs',
+          items: [
+            {
+              label: 'GettingStarted',
+              to: 'https://docs.pinot.apache.org/getting-started',
+            },
+            {
+              label: 'PinotComponents',
+              to: 'https://docs.pinot.apache.org/pinot-components',
+            },
+            {
+              label: 'UserGuide',
+              to: 'https://docs.pinot.apache.org/pinot-user-guide',
+            },
+            {
+              label: 'Administration',
+              to: 'https://docs.pinot.apache.org/operating-pinot',
+            },
+          ],
+        },
+        {
+          title: 'Community',
+          items: [
+            {
+              label: 'Slack',
+              to: 'https://communityinviter.com/apps/apache-pinot/apache-pinot',
+            },
+            {
+              label: 'Github',
+              to: 'https://github.com/apache/incubator-pinot',
+            },
+            {
+              label: 'Twitter',
+              to: 'https://twitter.com/ApachePinot',
+            },
+            {
+              label: 'Mailing List',
+              to: 'mailto:dev-subscribe@pinot.apache.org?Subject=SubscribeToPinot',
+            },
+          ],
+        },
+      ],
+      logo: {
+        alt: 'Apache Pinot™ - Incubating',
+        src: 'img/logo.svg',
+        href: 'https://pinot.apache.org/',
+      },
+      copyright: `Copyright © ${new Date().getFullYear()} The Apache Software Foundation.`,
+    },
+    googleAnalytics: {
+      // TODO
+      trackingID: 'TEMP',
```

Comment:
Please refer to `https://github.com/apache/incubator-pinot-site/pull/16/files`
Is this the tracking id 'UA-157446650-1'?